polkadex-primitives/src/lib.rs

```diff
@@ -6,22 +6,21 @@ use sgx_tstd as std;
 
 use std::vec::Vec;
 
-use codec::{Encode, Decode};
+use codec::{Decode, Encode};
 use sp_core::crypto::AccountId32;
 
-pub type AccountId = [u8;32];
+pub type AccountId = AccountId32;
```

Comment:
This is not AccountId32 but AccountId imported from polkadex-primitives of Polkadex Node

Reply:
Yes @Gauthamastro, as soon as that PR is merged ... I will replace it